### PR TITLE
Refactor CSS and dialog example

### PR DESCRIPTION
**Pull request by Hofer-Julian**
- Put CSS into separate file
- Rename and simplify dialog example a bit

### CODE REVIEW

File: examples/css/style.css [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+combobox button.combo box {
+    padding: 5px;
+}
+combobox box arrow {
+    -gtk-icon-source: none;
+    border-left: 5px solid transparent;
+    border-right: 5px solid transparent;
+    border-top: 5px solid black;
+}
```
Comment (sdroege) [inline examples/css/style.css:28]: Better to have a newline at the end

**Review by sdroege — COMMENTED** [review]
Looks good to me otherwise

File: examples/dialog_async/main.rs [hunk 2]
```diff
@@ -36,30 +20,41 @@ fn build_ui(application: &gtk::Application) {
         .visible(true)
         .build();
 
-    let window = Rc::new(
```
Comment (sdroege) [inline examples/dialog_async/main.rs:39]: The `Rc` here was very strange

File: examples/css/style.css (hunk 1 above)
Comment (Hofer-Julian) [reply examples/css/style.css]: Done.
Also added the setting to my editor, even though I never understood what difference it makes.

File: examples/dialog_async/main.rs (hunk 2 above)
Comment (Hofer-Julian) [reply examples/dialog_async/main.rs:39]: Yep...

I also found the image holder in cairo_threads a bit too over-engineered.
Maybe it should also be simplified
https://github.com/gtk-rs/gtk-rs/blob/master/examples/cairo_threads/image/image_holder.rs

Comment (Hofer-Julian) [reply examples/dialog_async/main.rs:39]: To be more explicit, remove the image_holder altogether and wrap `Image.data` in a Rc<Refcell<>>
https://github.com/gtk-rs/gtk-rs/blob/master/examples/cairo_threads/image/mod.rs

Comment (sdroege) [reply examples/dialog_async/main.rs:39]: Can you show what you're thinking of?

Comment (Hofer-Julian) [reply examples/dialog_async/main.rs:39]: Sure will open another PR

File: examples/css/style.css (hunk 1 above)
Comment (sdroege) [reply examples/css/style.css]: It make diffs more useful to read later at least :)

**Pull request merged**